core/schema.go

```diff
@@ -37,8 +37,9 @@ var (
 	}
 
 	SchemaForceNewOptional = &options{
-		Optional: true,
 		ForceNew: true,
+		Computed: true,
```

Comment:
Why is computed now needed?

Reply:
There are parameters in our APIs that are both `Computed` and `Optional` and they should be `ForceNew` as well for the update. Kind of the same logic as `SchemaComputedOptional`